apps/emqx_bridge_kinesis/src/emqx_bridge_kinesis.erl

```diff
@@ -55,6 +55,7 @@ fields(connector_config) ->
             mk(
                 binary(),
                 #{
+                    required => true,
                     default => <<"https://kinesis.us-east-1.amazonaws.com">>,
```

Comment:
As it is required, I think the default value will be useless. Maybe remove it or use it as example
```suggestion
                    required => true,
                    example => <<"https://kinesis.us-east-1.amazonaws.com">>,
```

Reply:
It looks right.